test/unit/lib/plugins/aws/package/compile/events/websockets/index.test.js

```diff
@@ -131,23 +133,34 @@ describe('test/unit/lib/plugins/aws/package/compile/events/websockets/index.test
             },
           },
         },
-      });
+      }));
     });
 
     it('should create a websocket api resource', () => {
+      expect(cfTemplate.Resources.WebsocketsApi).to.not.equal(undefined);
```

Comment:
Can we also confirm on generated resource content (in similar manner as test we replace did) ?

And same in below cases

Reply:
@medikoo I added a commit with content check now, but below cases did not have any checks like that (or am I missing something?) so I only changed the case starting on line 139. Thankful for feedback!

Reply:
@eli6 yes, it was just about those which test that